src/app/settings/settings/settings.component.spec.ts

```diff
@@ -1,37 +1,98 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import {
+  async,
+  ComponentFixture,
+  TestBed,
+  inject
+} from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
 
-import { CoreModule } from '@app/core';
+import { MatSlideToggle } from '@angular/material';
+
 import { SharedModule } from '@app/shared';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+
+import { By } from '@angular/platform-browser';
+
+import { Store } from '@ngrx/store';
```

Comment:
Also would be nice to order imports:

1. all @angular / library limports like ngrx
2. space
3. @core, @shared, @testing imports
4. space
5. relative imports './'